skrf/network.py

```diff
@@ -467,12 +485,12 @@ def from_z(cls, z: npy.ndarray, *args, **kw) -> 'Network':
             Network, `f` and `z0`.
 
         Return
```

Comment:
Sorry to annoy you, but I just noticed that it should be `Returns` (not `Return`) according to the NumPy convention (multiple occasions). Also there sometimes still is the old `:return:` above the new `Returns` statement.

Reply:
no problem, you are right. Since I'm on it, it's the right time to make all these small corrections !